dashboard/src/components/Config/AppRepoList/AppRepoButton.tsx

```diff
@@ -43,7 +43,7 @@ export class AppRepoAddButton extends React.Component<
   public render() {
     const { redirectTo } = this.props;
     return (
-      <div className="AppRepoAddButton">
+      <span>
```

Comment:
why did you change this for a span?

Reply:
If using <div>, the two buttons will show in two lines.

Reply:
I see, in this case then it's better to use a component call `<React.Fragment>` that allows you to group elements

Reply:
I found that the 'Delete' and 'Refresh' buttons in AppRepoListItem.tsx simply use two `<button>` in a `<td>`. How about using `<span>` here or remove `<span>`?

Reply:
We should try to use something other than `span`, since `span` is an inline element and shouldn't really be wrapping buttons. You might be able to use flexbox (https://design.bitnami.com/category/Foundations/Helpers/#Flex%20Align) in the AppRepoList to make these buttons show up next to each other, but if you aren't familiar with that then I would also suggest trying to use `React.Fragment`.

We can't just use a button element here because of the Modal.

Reply:
Thanks. Let me try `React.Fragment`.

Reply:
Done. Now using `React.Fragment`.